constance/apps.py

```diff
@@ -1,45 +1,9 @@
-from django.db.models import signals
-from django.apps import apps, AppConfig
+from django.apps import AppConfig
 from django.utils.translation import gettext_lazy as _
 
 
 class ConstanceConfig(AppConfig):
     name = 'constance'
     verbose_name = _('Constance')
+    default_auto_field = 'django.db.models.AutoField'
```

Comment:
```suggestion
    default_auto_field =getattr(settings, "DEFAULT_AUTO_FIELD", 'django.db.models.AutoField')
```

why not this could be overrrited by a variables from setting.

Reply:
Do you mean default django settings or project settings? In both cases things can change and generate extra migration in installed dependencies what is not  expected